concent_api/core/views.py

```diff
@@ -311,10 +396,15 @@ def handle_receive_ack_from_force_report_computed_task(decoded_message):
     return ack_report_computed_task
 
 
-def handle_receive_force_get_task_result_upload_for_provider(request, decoded_message: message.concents.ForceGetTaskResult) -> message.concents.ForceGetTaskResult:
+def handle_receive_force_get_task_result_upload_for_provider(
+    request,
+    decoded_message:                message.concents.ForceGetTaskResult,
+    previous_message_from_database: Message
```

Comment:
It's not a `Message`. It's a message status.

Also, this looks really brittle to me. It's easy to give the wrong message here. You should add some asserts. Is ` previous_message_from_database` the same message the `decoded_message` was decoded from? You should check that.

I think it would make more sense to just pass the key here, not the whole status object. Less things that can go out of sync that way. It's not really clear by just looking at the name what `previous_message_from_database` is supposed to be.

Reply:
The request is used in some of the function for other things, so it is better to keep their signature consistent and pass request to every one that is using it.

Reply:
I'm not talking about `request`. I'm talking about `previous_message_from_database` which is only used to get the key out of it.